pgbackrest_auto

```diff
@@ -464,6 +466,7 @@ function pgbackrest_exec(){
     if bash -c "pgbackrest --config=${backup_conf} --repo1-host=${BACKUPHOST} --repo1-host-user=postgres --stanza=${FROM} --pg1-path=${TO} ${pgbackrest_opt} --delta restore --process-max=4 --log-level-console=error --log-level-file=detail --recovery-option=${recovery_opt} --tablespace-map-all=${TO}/remapped_tablespaces"
     then
         info "Restore from backup done"
+        sed -i 's/Restore_from_backup=0/Restore_from_backup=1/g' ${zbx_status_file}
```

Comment:
why is the status not written when a restore error occurs?

Reply:
before new restore all values resets in zabbix status file (line 602)